uber/site_sections/utility.py

```diff
@@ -0,0 +1,63 @@
+from uber.common import *
+import subprocess
+import shlex
+
+# admin utilities.  should not be used during normal ubersystem operations except by developers / sysadmins
+
+
+# quick n dirty. don't use for anything real.
```

Comment:
This is the kind of comment which makes me leery of merging this into ``master`` on the core repo.  I'd feel a lot more comfortable if this was going into ``magprime``.